.github/workflows/release.yml

```diff
@@ -27,6 +29,12 @@ jobs:
       with:
         username: ${{ secrets.DOCKER_HUB_USER }}
         password: ${{ secrets.DOCKER_HUB_TOKEN }}
+    - uses: docker/login-action@v2
+      if: startsWith(github.ref, 'refs/tags/')
```

Comment:
I think both of this line and 28th line can be deleted

Reply:
Updated

Reply:
Updated @Rory-Z